src/main/java/org/springframework/integration/kafka/support/KafkaHeaders.java

```diff
@@ -28,8 +29,12 @@
 
 	public static final String MESSAGE_KEY = PREFIX + "messageKey";
 
-	public static final String PARTITION_ID = PREFIX + "_partitionId";
+	public static final String PARTITION_ID = PREFIX + "partitionId";
 
-	public static final String OFFSET = PREFIX + "_offset";
+	public static final String OFFSET = PREFIX + "offset";
```

Comment:
To be honest, I'd move to 1.1 and get done with it at all. This and retriability are big enough features to warrant a 1.1 I believe.